packages/@blockprotocol/type-system-node/test/data-type.test.ts

```diff
@@ -51,7 +51,7 @@ const primitiveDataTypes: DataType[] = [
 const invalidDataTypes: DataType[] = [
   {
     kind: "dataType",
-    $id: "https://blockprotocol.org/@blockprotocol/types/data-type/text/", // unversioned URI
```

Comment:
Wouldn't it also make sense to keep an invalid data type that uses an unversioned URI for testing?